src/common/Header/Search/index.js

```diff
@@ -1,22 +1,35 @@
 import { SearchIcon, SearchInput, SearchWrapper } from "./styled";
+import { toPeopleList, toMoviesList } from "../../../core/routes";
 import search from "./search.svg";
 import {
   useQueryParameter,
   useReplaceQueryParameter,
 } from "../../../features/queryParameters";
 import searchQueryParamName from "../../../features/searchQueryParamName.js";
 import { useLocation } from "react-router";
+import { useState } from "react";
 
 const Search = () => {
-  const query = useQueryParameter(searchQueryParamName);
+  const queryURL = useQueryParameter(searchQueryParamName);
   const replaceQueryParameter = useReplaceQueryParameter();
   const location = useLocation();
+  const [query, setQuery] = useState(queryURL);
+  const [wait, setWait] = useState(true);
```

Comment:
Perhaps it would be better to call the state isWait or isFetching instead wait?

Reply:
Good idea, isWait sounds better :)